alleycats-core/src/test/scala/alleycats/SyntaxSuite.scala

```diff
@@ -54,6 +54,21 @@ object SyntaxSuite {
     val a1: Boolean = x.nonEmpty
   }
 
+  def testOptionEmpty[A: Empty]: Unit = {
+    case class A[T](x: T)
+    case class B()
+
+    import alleycats.std.option.*
+    implicit def emptyA[T: Empty]: Empty[A[T]] = new Empty[A[T]] {
+      def empty: A[T] = A(Empty[T].empty)
+    }
+    implicit val emptyB: Empty[B] = new Empty[B] {
```

Comment:
You don't need this do you? Shouldn't we have an empty `Option[B]` for all `B`?

Reply:
That right. I pushed